### PR TITLE
[Components with more than 1 service] LPS-204299 Refactor FragmentEntryLinkModelListener component

### PR DESCRIPTION
**Pull request by rafaprax**
**Author:** @regiCesar21
**Issue:** https://liferay.atlassian.net/browse/LPS-204299
**Forwarded from:** https://github.com/rafaprax/liferay-portal/pull/1399
**Original comment:**
ticket: https://liferay.atlassian.net/browse/LPS-204299
test made using same case as: https://liferay.atlassian.net/browse/LPS-175207

### CODE REVIEW

**Comment by regiCesar21** [comment]
ci:test:clustering

**Comment by rafaprax** [comment]
ci:test:clustering

**Comment by rafaprax** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-renderer-react-impl/src/main/java/com/liferay/fragment/renderer/react/internal/model/listener/FragmentEntryLinkModelListener.java [hunk 1]
```diff
@@ -191,9 +206,10 @@ private void _updateNPMRegistry(
 	private static final Log _log = LogFactoryUtil.getLog(
 		FragmentEntryLinkModelListener.class);
 
+	private static BundleContext _bundleContext;
```
Comment (dantewang) [inline modules/apps/fragment/fragment-renderer-react-impl/src/main/java/com/liferay/fragment/renderer/react/internal/model/listener/FragmentEntryLinkModelListener.java:209]: @rafaprax Please just use a Snapshot to simplify the related logic.

**Comment by dantewang** [comment]
Plz resend, thanks!

File: modules/apps/fragment/fragment-renderer-react-impl/src/main/java/com/liferay/fragment/renderer/react/internal/model/listener/FragmentEntryLinkModelListener.java (hunk 1 above)
Comment (rafaprax) [reply modules/apps/fragment/fragment-renderer-react-impl/src/main/java/com/liferay/fragment/renderer/react/internal/model/listener/FragmentEntryLinkModelListener.java:209]: cc @regiCesar21

**Pull request closed without merging**